lm_admin_guide.md

```diff
@@ -323,52 +323,99 @@ required by the `psycopg2` Python package.
+3. upgrade the database schema by typing:
+
     ```bash
-    docker cp life_monitor_db_1:/tmp/lifemonitor_backup.sql ${HOME}/lifemonitor_backup.sql
+    flask db upgrade
     ```
-4. teardown all the services in `prod` mode
+
+4. restart the LifeMonitor Flask app.
+
+##### Deployment based on `docker-compose`
+
+Existing deployments based on `docker-compose` can be easily upgraded to a more recent LifeMonitor version by going through the following steps:
+
+1. teardown all the services in `prod` mode
+
     ```bash
     make down
     ```
-5. update your local copy of LifeMonitor sources (via `git clone` or `git pull`)
-6. restart all the services 
+2. update your local copy of LifeMonitor sources (via `git clone` or `git pull`)
+3. restart all the services
     ```bash  
     make start
     ```
     or type `make start-dev` to restart all the services in `dev` mode.
 
+> **NOTE.** As part of the initialisation process performed by the `init` container, the database is automatically upgraded when the docker-compose app restarts. So, you don't need to perform the upgrade manually.
 
-As a result, the up-to-date LifeMonitor instance will be started and the existing data will be migrated to the proper database schema. 
+##### Check current schema version
 
 You can always check the actual running database schema by typing:
 
 ```bash
 docker-compose exec lm /bin/bash -c "flask db current"
 ```
 
-> An output with `(head)` at the end, e.g., `bbe1397dc8a9 (head)`, indicates that your LifeMonitor instance is running with the most recent database schema.
+> An output with `(head)` at the end - e.g., `bbe1397dc8a9 (head)` - indicates that your LifeMonitor instance is running with the most recent database schema.
 
-### Upgrade a non-Docker deployment
-To upgrade a LifeMonitor instance deployed without Docker (see section on ["How to install on your local environment"](#how-to-install-on-your-local-environment)), you have to:
-1. stop LifeMonitor Flask app;
-2. make a backup of the database used by LifeMonitor;
-3. update your local copy of LifeMonitor sources (via `git clone` or `git pull`);
-4. apply all the required migrations, by typing 
-    ```bash
-    flask db upgrade
-    ```
-5. restart the LifeMonitor Flask app.
+
+
+## How to backup and restore
+
+The current implementation of LifeMonitor relies on **PostgreSQL** database management system. Thus, you can use the common `pg_dump` and `psql` tools provided with Postgres to respectively backup and restore the LifeMonitor database (see [PostgreSQL docs](https://www.postgresql.org/docs/11/backup.html) for more details).
+
+### Backup
+
+To make a backup to file (e.g., `lifemonitor.sql`) type:
+```bash
+pg_dump -U <POSTGRESQL_USERNAME> <POSTGRESQL_DATABASE> > lifemonitor_backup.sql
+```
+
+If you are using the PostgreSQL instance of the [`docker-compose` LifeMonitor deployment](#deploy-Lifemonitor-with-docker-compose), go through the following stpes:
```

Comment:
This link does not seem to be working (checked at https://kikkomep.github.io/life_monitor/lm_admin_guide#Deploy-LifeMonitor-with-docker-compose)